README.md

```
@@ -271,6 +309,10 @@ docker run -p 9010:9010 minio/hperf:latest server --address 0.0.0.0:9010
 **Symptom**: Unusually high throughput or low latency results
 **Solution**: Ensure `--real-ip` matches the external IP used for inter-server communication
 
+### Server exits with "unable to listen on ..."
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)